chapters/data-collection.tex

```diff
@@ -1,362 +1,299 @@
+and loss to follow-up occurred in the field and how they were implemented and resolved.
+
+High frequency checks should also include survey-specific data checks. As electronic survey 
+software incorporates many data control features, discussed above, these checks 
+should focus on issues survey software cannot check automatically. As most of 
+these checks are survey specific, it is difficult to provide general guidance. 
+An in-depth knowledge of the questionnaire, and a careful examination of the 
+pre-analysis plan, is the best preparation. Examples include  consistency 
+across multiple responses, complex calculations, suspicious patterns in survey 
+timing, or atypical response patters from specific enumerators. 
+\sidenote{\url{https://dimewiki.worldbank.org/wiki/Monitoring_Data_Quality}} 
+survey software typically provides rich metadata, which can be useful in 
+assessing interview quality. For example, automatically collected time stamps 
+show how long enumerators spent per question, and trace histories show how many 
+times answers were changed before the survey was submitted.
+
+
+\subsection{Data considerations for field monitoring}
+Careful monitoring of field work is essential for high quality data.
+\textbf{Back-checks}\sidenote{\url{https://dimewiki.worldbank.org/wiki/Back_Checks}} and other survey audits help ensure that enumerators are following established protocols, and are not falsifying data.
+For back-checks, a random subset of the field sample is chosen and a subset of information from the full survey is verified through a brief interview with the original respondent.
+Design of the back-check questionnaire follows the same survey design 
+principles discussed above: you should use the pre-analysis plan 
+or list of key outcomes to establish which subset of variables to prioritize.
+
+Real-time access to the survey data increases the potential utility of 
+back-checks dramatically, and both simplifies and improves the rigor of related 
+workflows.
+You can use the raw data to draw the back-check sample; assuring it is 
+appropriately apportioned across interviews and survey teams.
+As soon as back-checks are complete, the back-check data can be tested against 
+the original data to identify areas of concern in real-time.
+\texttt{bcstats} is a useful tool for analyzing back-check data in Stata module.
+\sidenote{\url{https://ideas.repec.org/c/boc/bocode/s458173.html}}
+
+Electronic surveys also provide a unique opportunity to do audits through audio recordings of the interview, typically short recordings triggered at random throughout the questionnaire.
+\textbf{Audio audits} are a useful means to assess whether the enumerator is conducting the interview as expected (and not sitting under a tree making up data).
+Do note, however, that audio audits must be included in the Informed Consent.
+
+\textcolor{red}{
+\subsection{Dashboard}
+Do we want to include something here about displaying HFCs? }
```

Comment:
I think this chapter is already long enough without including a new subsection. We could maybe add a line when we suggest using `ipacheck` that says something like

> `ipacheck` generates an excel sheet with results for each run. It is common for teams to choose other formats to explore results, notably online dashboards. However, their creation involves writing custom scripts.